cmd/kube-apiserver/apiserver.go

```diff
@@ -51,12 +51,8 @@ var (
 		"further assumed that port 443 on the cluster's public address is proxied to this "+
 		"port. This is performed by nginx in the default setup.")
 	address               = util.IP(net.ParseIP("127.0.0.1"))
-	publicAddressOverride = flag.String("public_address_override", "", ""+
-		"Public serving address. Read only port will be opened on this address, "+
-		"and it is assumed that port 443 at this address will be proxied/redirected "+
-		"to '-address':'-port'. If blank, the address in the first listed interface "+
-		"will be used.")
-	readOnlyPort = flag.Int("read_only_port", 7080, ""+
+	publicAddressOverride = util.IP(net.ParseIP("127.0.0.1"))
```

Comment:
You have introduced a semantically meaningful change.  Before this change, publicAddressOverride defaulted to "", which was caught in master.go, and assigned to an auto-selected interface.  After this change, publicAddressOverride defaults to 127.0.0.1, but master.go looks for nil.

I predict that this will fail e2e with at least the DNS test failing.

If I am right, try simply initializing this as util.IP{}, which should match nil, which should retain the semantic.

But i actually want you to run e2e.